lxd/api_1.0.go

```diff
@@ -211,6 +211,12 @@ func api10Get(d *Daemon, r *http.Request) Response {
 		"shiftfs":            fmt.Sprintf("%v", d.os.Shiftfs),
 	}
 
+	env.LXCFeatures = map[string]string{
```

Comment:
The various features in there should be checked at startup and cached as `lxc.HasApiExtension` is pretty costly and `/1.0` is our most queried API endpoint.